testing/btest/language/line-numbers.zeek

```diff
@@ -0,0 +1,12 @@
+# @TEST-DOC: Test for regression in printing line numbers out-of-order
+#
+# Older versions of Zeek would sometimes print statement line numbers
+# reversed (like "lines 49-48"). The issue most noticeably occurs in
+# script profiling, so this test uses script profiling to look for any
+# instances.
+# @TEST-EXEC: ZEEK_PROFILER_FILE=test.prof zeek -b /dev/null
+# @TEST-EXEC: grep 'lines [0-9]*-[0-9]' test.prof >multi-lines
```

Comment:
Should that be more like `grep 'lines [0-9]+-[0-9]+' test.prof >multi-lines` ?

I find the test pretty opaque the empty output is probably easy to produce when the regex isn't matching anything or the awk script doesn't do the right thing ;-)

I wonder if an `InternalError()` or `InternalWarning()` in `Location::Describe()` is sufficient when an inconsistency is found. What'd be your reaction to that?